core/primitives/src/crypto.rs

```diff
@@ -758,7 +770,10 @@ pub trait Pair: CryptoType + Sized + Clone + Send + Sync + 'static {
 	/// be equivalent to no password at all.
 	///
 	/// `None` is returned if no matches are found.
-	fn from_string_with_seed(s: &str, password_override: Option<&str>) -> Result<(Self, Option<Self::Seed>), SecretStringError> {
+	#[cfg(feature = "std")]
+    fn from_string_with_seed(s: &str, password_override: Option<&str>) 
```

Comment:
Spaces in front of the function name

Reply:
done. hopefully there's only this one occurrence.